containers/unit_tests/TestUnorderedMap.hpp

```diff
@@ -158,10 +157,8 @@ struct TestErase {
   TestErase(map_type map, uint32_t num_erases, uint32_t num_duplicates)
       : m_map(map), m_num_erase(num_erases), m_num_duplicates(num_duplicates) {}
 
-  void testit() {
-    execution_space().fence();
-    Kokkos::parallel_for(m_num_erase, *this);
-    execution_space().fence();
+  void testit(const execution_space& exec) {
+    Kokkos::parallel_for(Kokkos::RangePolicy<execution_space>(exec, 0, m_num_erase), *this);
```

Comment:
All the other `tesit`s fence the execution space instance.
```suggestion
    Kokkos::parallel_for(Kokkos::RangePolicy<execution_space>(exec, 0, m_num_erase), *this);
    exec.fence();
```